File: Auto-generated-tasks/Difficulty-2/Task-4/Solution.f90
integer:: Cnt = 5, k1, Rez = 0
Сharacter(len=Kol):: Tabl
do k1 = 0, Cnt
	if (T(k1) < 186) then
		Rez = Rez + 1;
	end if;
end do
print *, "Rez=", Rez

File: Auto-generated-tasks/Difficulty-2/Task-4/Solution.pas
Var
	Cnt:=5, k1, Rez:=0:integer;
	T:Array[0..Cnt] of integer;
Begin
	for k1:= 0 to Cnt do begin
		if (T[k1] < 186) then begin
			Rez := Rez + 1;
		end;
	end;
	WriteLn("Rez=", Rez);
End.